### DISCUSSION TITLE
Chat popouts aren't classed as a window

### DISCUSSION THREAD

**Discussion by TigerWarz** [Chatterino/chatterino2#3881] [Ideas] [answered]
When popping out a chat, it is seen as a part of the main chatterino window, meaning that to be able to see the popout you have to have the main chatterino window focussed.

I feel like it would be better if it was classed as its own window, so for example you could have the chat you are currently in as a popout, but keep the others open for mentions, etc.

Small request, and I believe it may have been like this in the past? I don't exactly know though

**Comment by Felanbird** [comment]
It's a bug - #3863

**Comment by 8thony** [comment, marked as answer]
- Fixed since #4181
